### DISCUSSION TITLE
Papers about LaTeX-OCR, want to learn more principles and innovate

### DISCUSSION THREAD

**Discussion by XiaoyangLiu-sjtu** [lukas-blecher/LaTeX-OCR#213] [Ideas]
Dear lukas:

Long time no see, I used your project about a year ago when I was working on an image recognition project, and you solved a lot of my problems at that time. After a year, I am very happy to see that your project is getting better and better, and the attention is also increasing. At present, I am in the stage of graduation thesis, and I am very interested in LaTeX-OCR. I would like to ask you, are there any papers that can learn related principles? I want to use LaTeX-OCR as my graduation thesis research direction and try to innovate and improve, or even Postgraduate research direction.

Any help and response will be of great help, thanks！

Best wishes,
Xiaoyang Liu

**Comment by lukas-blecher** [comment]
Hi, thanks for the encouraging words.

Besides the papers linked in the README I found the following:
- Mathematical Formula Identification in PDF Documents, Lin (2011) http://www.iapr-tc11.org/archive/icdar2011/fileup/PDF/4520b419.pdf
- Translating Math Formula Images to LaTeX Sequences Using Deep Neural Networks with Sequence-level Training Zelun Wang (2019) https://arxiv.org/abs/1908.11415
- A Transformer-based Math Language Model for Handwritten Math Expression Recognition (2021) https://arxiv.org/abs/2108.05002

More general but relevant
- OCR-free Document Understanding Transformer (2021) https://arxiv.org/abs/2111.15664
- TrOCR: Transformer-based Optical Character Recognition with Pre-trained Models (2021) https://arxiv.org/abs/2109.10282
- DocFormer: End-to-End Transformer for Document Understanding (2021), https://arxiv.org/abs/2106.11539

If you find any more I'd be happy to hear about them.
Good luck!
